akka-http-core/src/test/scala/akka/http/scaladsl/GracefulTerminationSpec.scala

```diff
@@ -55,7 +55,7 @@ class GracefulTerminationSpec
       // immediately trying a new connection should cause `Connection refused` since we unbind immediately:
       val r3 = makeRequest(ensureNewConnection = true)
       val ex = intercept[StreamTcpException] {
-        Await.result(r3, 2.seconds)
+        Await.result(r3, 5.seconds)
```

Comment:
Hold on, this really shouldn't be necessary, seems the config is different after all. looking into it.